### PR TITLE
[FIX] stock_inventory_import: Return in action_done

### PR DESCRIPTION
**Pull request by hugosantosred**
The action_done method in stock_inventory_import must return True, because if None is returned it could cause problems with xmlrpc method calls.

### CODE REVIEW

File: stock_inventory_import/models/stock_inventory.py [hunk 1]
```diff
@@ -84,3 +84,4 @@ def action_done(self):
                     _("Loaded lines must be processed at least one time for "
                       "inventory : %s") % (inventory.name))
             super(StockInventory, inventory).action_done()
+        return True
```
Comment (pedrobaeza) [inline stock_inventory_import/models/stock_inventory.py]: Better to return the value of super

**Comment by pedrobaeza** [comment]
Please check Travis.

**Comment by hugosantosred** [comment]
@pedrobaeza Travis is passing now, without changing the code. The test that was failing in travis is from module product_price_history.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**